src/components/Button/MuiButton.tsx

```diff
@@ -0,0 +1,123 @@
+import Button, { ButtonProps as MuiBtnProps } from '@mui/material/Button';
+import React, { forwardRef } from 'react';
+
+import { styled } from '@mui/material';
+import Loader from '../Icons/Loader';
+import styles from './Button.module.scss';
+import { tokens } from '../../theme/tokens';
+
+export interface MuiButtonProps extends Omit<MuiBtnProps, 'size'> {
+  loading?: boolean;
```

Comment:
is this something we keeping?

Reply:
yeah.
MUI Button does not have a loader except in the experimental lab version
https://mui.com/material-ui/react-button/#loading-button

Reply:
but do we need a loading state? did you asked Elsa?

Reply:
asked her and she said it works for now